lib/klaviyo.js

```diff
@@ -75,6 +77,18 @@ class Klaviyo {
     set lists(value) {
         throw new KlaviyoError(WRITE_ACCESS_ERROR);
     }
+
+    get dataprivacy() {
```

Comment:
I think this should be camal cased
```suggestion
    get dataPrivacy() {
```